frontend/packages/component-library/src/actionBlock/actionBlock.module.scss

```
@@ -0,0 +1,73 @@
+@use '@code-dot-org/component-library-styles/mixins.scss' as mixins;
+
+$mobileBreakpoint: 900px;
+
+// ActionBlock common styles
+.actionBlock {
+  @include mixins.regular-border-styles;
```

If we will use my suggestion, then here it will simply be:
```suggestion
  @include mixins.border-styles();
```